examples/binary-bitwise-expressions/binary_bitwise_expressions.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/io;
+import ballerina/lang.'int as ints;
+
+public function main() {
+    int a = 385;
+    ints:Unsigned8 b = 128;
+    // For a bitwise AND (`&`) expression, if the type of either operand is a subtype of
+    // `int:UnsignedK` when K is 8, 16 or 32, then the type of the result is `int:UnsignedN`
+    // where N is the smallest such K.
+    ints:Unsigned8 res1 = a & b;
+    io:println("`int` 385 & `ints:Unsigned8` 128: ", res1);
+
+    ints:Signed16 c = -32700;
+    int d = 249;
+    // Else, if neither operand is of a type that is a subtype of an unsigned `int` type,
+    // the type of the result for a bitwise AND expression is `int`.
```

Comment:
Same comment as above re: referencing to bitwise AND as an expression. Can see in other places too